src/screens/activity/events.screen.js

```diff
@@ -79,25 +80,23 @@ const styles = StyleSheet.create({
 
 class Events extends Component {
   componentDidMount() {
-    const { user: { login }, getUser } = this.props;
+    const { getEvents, user: { login } } = this.props;
 
-    if (login) {
-      this.getUserEvents();
-    } else {
-      getUser();
```

Comment:
Maybe I missed something, there are too many changes, but did we get the `user` in advance? The point is that this check is needed when we just enter the application (when only logged in), we do not have the `user` (because of lazy loading tabs), so we get data on it, and only then we receive events for it.

Reply:
Absolutely buddy, I have a TODO comment for that line 95.

I'm thinking about moving the getUser() call to the "Preparing GitPoint.." screen right after the login. There is no point in opening the App if that special call fail, and we don't have `getState().auth.user`.

 What do you think about that?

Reply:
Yeah, totally agree :+1: